irc_im.c

```diff
@@ -969,6 +969,31 @@ static gboolean bee_irc_channel_chat_invite( irc_channel_t *ic, irc_user_t *iu )
 	return TRUE;
 }
 
+static void bee_irc_channel_chat_kick( irc_channel_t *ic, irc_user_t *iu, const char *msg )
+{
+	struct groupchat *c = ic->data;
+	bee_user_t *bu = iu->bu;
+	
+	if( bu == NULL )
+		return;
+	
+	if( c && ( iu->bu->ic != c->ic ) )
+	{
+		irc_send_num( ic->irc, 482, "%s :Can't mix different IM networks in one groupchat", ic->name );
+		return;
+	}
+	else if( !c->ic->acc->prpl->chat_kick )
+	{
+		irc_send_num( ic->irc, 482, "%s :IM protocol does not support room kicking", ic->name );
+		return;
+	}
+	
+	/* TODO: Need more const goodness here, sigh */
```

Comment:
IMO, yes, be constant. Turning the others into constants wouldn't hurt either.

Is this the reason you did the `g_strdup()` below? Because in that case, constants all the way.